templates/interface/filter-panels.interface.hbs

```diff
@@ -58,9 +58,26 @@
         </div>
       </div>
 
+      <div class="form-group clearfix" id="filter-value-type">
+        <div class="col-sm-4 control-label">
+          <label>Value type</label>
+        </div>
+        <div class="col-sm-8">
+          <label for="value-type-field-{{id}}" class="select-proxy-display">
+            <select name="value-type-field-{{id}}" id="value-type-field-{{id}}" data-field="valueType" data-label="-- Enter a value" class="hidden-select form-control">
```

Comment:
What's `data-label` for? https://developers.fliplet.com/UI-guidelines-interface.html#dropdown-list doesn't specify it needs it. Does this component have a specific need for it?

Reply:
@tonytlwu fixed, because we have the default value 'enter-value' to `data-label` not needed

Reply:
@tonytlwu fixed, because we have the default value 'enter-value' to data-label not needed